recipes/rasterstats/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set version = "0.10.3" %}
+
+package:
+    name: rasterstats
+    version: {{ version }}
+
+source:
+    fn: rasterstats-{{ version }}.tar.gz
+    url: https://pypi.io/packages/source/r/rasterstats/rasterstats-{{ version }}.tar.gz
+    sha256: 7a4a79d71fbfb15812ee40081a8fd321039e9b26f389f7ec456c657c3298a818
+
+build:
+    number: 0
+    preserve_egg_dir: True
+    script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+    build:
+        - python
+        - setuptools
+    run:
+        - python
+        - shapely
+        - numpy >=1.9
+        - rasterio >=0.27
+        - cligj >=0.4
+        - fiona
+        - gdal 1.11.4
+
+test:
+    imports:
+        - rasterstats
+    requires:
+        - coverage
+        - pyshp >=1.1.4
+        - pytest
+
+about:
+  home: https://github.com/perrygeo/python-raster-stats
+  license: BSD License
+  summary: 'Summarize geospatial raster datasets based on vector geometries.'
+
+extra:
+    recipe-maintainers:
+        - perrygeo
```

Comment:
Certainly. I'm not a conda user myself but this would be a good way for me to understand the packaging process.